Cargo.toml

```
@@ -297,6 +297,9 @@ name = "ring"
 [dependencies]
 untrusted = "0.6.2"
 
+[target.'cfg(not(target_os = "ios"))'.dependencies]
+spin = { version = "0.5.0", default-features = false }
```

```suggestion
spin = "0.4.10"
```
This would support @baidu/rust-sgx-sdk and there are no features in spin so that disabling default-features isn't needed

---

With this:
```toml
[target.'cfg(not(target_os = "ios"))'.dependencies]
spin = { version = "0.4.10", features = ["once"] }
```

I get:
```
error[E0554]: #![feature] may not be used on the stable release channel
#![cfg_attr(feature = "const_fn", feature(const_fn))]
```

So we have to use spin 0.5 in order for it to work on the stable channel.

IMO it would be much better to have the Baidu SGX SDK update the version of Rust it uses to a newer version. Is that not going to happen any time soon?

---

That's too bad.
I hope it will be soon
https://github.com/baidu/rust-sgx-sdk/issues/48

---

So this isn't relevant anymore, Sorry for the trouble.
I still don't think the `default-features = false`, I see no features in the toml: https://github.com/mvdnes/spin-rs/blob/master/Cargo.toml

---

> I still don't think the `default-features = false`, I see no features in the toml: https://github.com/mvdnes/spin-rs/blob/master/Cargo.toml

Thanks! I updated the PR to remove `default-features = false`.